### DISCUSSION TITLE
Support for non-LTS .NET releases

### DISCUSSION THREAD

**Discussion by alanssitis** [dotnet/dotNext#215] [Ideas]
Hi dotNext maintainers!

I am a new-ish C# developer looking to integrate this library into my company's code base. One issue I've found is that you only support LTS versions at the moment, we migrate to the next version yearly, which makes us less likely to use dotNext. Would you be open to contributions to support non-LTS versions, and if so, show me the ropes?

**Comment by sakno** [comment]
Hi @alanssitis , what the goal you trying to achieve? For instance, .NET 6 dependency can be used without any problem from .NET 7 project. Currently, dotNext 5.x supports .NET 8 natively. In the time of .NET 5 I tried to produce multi-target release (3.1 and 5) but the experience was not good for many reasons:
* Hard to maintain and test code for both targets because of multiple `#if .. #endif`
* Dependency hell when you trying to consume 3.1 target from .net 5 project (MSBuild automatically selects the latest target version)

**Reply by alanssitis** [reply]
That makes sense; I am not too familiar with the extent of breaking APIs between major .NET versions. Therefore, I assumed that dotNext 4.X did not work on .NET 7 (I just learned about future versions of .NET being backward compatible).

I'll close the discussion. Thanks for the clarification!
